### ISSUE TITLE
Update tokenuuid to 2.0.2

### ISSUE THREAD

**Issue by theMusician** [esmero/archipelago-deployment-live#53]
Following the chats in Slack, it seems that Token UUID could be upgraded to a supported release for the 1.0.0 release?

https://github.com/esmero/archipelago-deployment-live/blob/ac2a2155256466cf8fdd06c227e0415bd9471487/drupal/composer.default.json#L74

$ composer require 'drupal/tokenuuid:^2.0'

[2.0.2 is the latest release available](https://www.drupal.org/project/tokenuuid/releases/2.0.2). The change from 1.x to 2.x seems to mainly be marking as D9 compatible.

**Comment by DiegoPino** [comment]
Yes, totally! A new composer.json (default) and a lock are on the way. Will add a reference to this issue when done. Thanks!

**Comment by theMusician** [comment]
Thanks Diego.
